monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/src/utils/apis/ResourceAPI.jsx

```diff
@@ -138,4 +137,22 @@ export default class ResourceAPI {
         };
         return this.getHTTPClient().post(`/endpoints`, payload);
     }
+
+    addNewUser(userId, password, isAdmin) {
+
+        var payload = {
+            userId: userId,
+            password: password,
+            isAdmin: isAdmin
+        };
+        return this.getHTTPClient().post(`/users`, payload);
+    }
+
+    deleteUser(userId) {
+        return this.getHTTPClient().delete(`/users/${userId}`)
+    }
+
+    getUserById(userId) {
```

Comment:
Don't we need to add the method to getUsers by pattern and role?

Reply:
resolving as the offline discussion.